### DISCUSSION TITLE
Gist sync

### DISCUSSION THREAD

**Discussion by v-c0de** [massCodeIO/massCode#83] [Ideas]
What about sync with Gist (or other)? 

Example like Code Notes (https://lauthieb.github.io/code-notes/)

Handles public and private:
![image](https://user-images.githubusercontent.com/65813722/169690776-c65b8185-9d3e-4c84-ac26-897db9bec889.png)

**Comment by adildg** [comment]
That would be amazing 👍🏻

**Comment by y9domains** [comment]
That will be more convenient

**Comment by LucasZapico** [comment]
I use [lepton](https://github.com/hackjutsu/Lepton) but features haven't been enhanced for a while. A gists sync would be my number one feature request for Masscode... definitely thought this was a feature. 😌

**Reply by BenjaminHoegh** [reply]
yeah, ended up doing the same. A shame there aren't any good solutions for windows users
